docs/usage/configuration-options.md

```diff
@@ -1845,6 +1845,30 @@ For example to apply a special label for Major updates:
 }
 ```
 
+### overwriteSourceUrl
+
+Use this field to set the source URL for release notes retrieval.
+Most commonly used for dependencies missing one.
+
+Using this field we can specify the exact url to fetch release notes from.
+
+Example setting source URL for package "dummy":
+
+```json
+{
+  "packageRules": [
+    {
+      "matchPackageNames": ["dummy"],
+      "overwriteSourceUrl": "https://github.com/org/dummy"
+    }
+  ]
+}
+```
+
+<!-- prettier-ignore -->
+!!! note
+Supported platforms are GitHub and GitLab only.
```

Comment:
You mean that we support changelogs on github/gitlab only, right? i.e. someone using platform=azure could still configure this for dependencies which are hosted on github.com

Reply:
correct, 

![image](https://user-images.githubusercontent.com/97394622/184083481-32bbb98e-1c4d-4c0c-b1d9-2bfb645bdfec.png)

and we overwrite it before above block